app/Http/Controllers/UserController.php

```diff
@@ -162,21 +153,6 @@ public function destroy(Request $request, $id)
             $user->delete();
         }
 
-        return redirect('user');
-    }
-    
-    /**
-     * Confirms deletion a user.
-     *
-     * @param  Request  $request
-     * @param  string  $id
-     * @return Response
-     */
-    public function remove(Request $request, $id)
-    {
-        $user = User::findOrFail($id);
-        $user->password = "";
-        
-        return view('user.remove', [ 'user' => $user ]);
+        return redirect('user.index');
```

Comment:
Redirection leads to page not found. You should replace this redirection with something like this:
```php
return redirect()->route('user.index')
    ->with('success', 'User deleted successfully');
```

Reply:
Fixed in c55e090